### ISSUE TITLE
Random Loading Phrase Selector Crashing

### ISSUE THREAD

**Issue by AEFeinstein** [fofix/fofix#199]
The code that selects a random loading phrase is crashing whenever a song is loaded in Quickplay. I'm no Python developer, but it seems that `random.choice()` doesn't exist, or maybe the wrong `random` is being used? I verified with the debugger that `self.engine.theme.loadingPhrase` is a valid list of strings.

![image](https://user-images.githubusercontent.com/231180/61098366-b98c3280-a42c-11e9-9717-c74879a3c926.png)

I have temporarily fixed this by replacing

```
if self.phrase == "":
	self.phrase = random.choice(self.engine.theme.loadingPhrase)
```
with
```
if self.phrase == "":
	self.phrase = "Whatever" # random.choice(self.engine.theme.loadingPhrase)
```

**Comment by Linkid** [comment]
Hi ! Thank you for this issue !
I had it last time. I don't think it's python because the ``random.choice`` exists : https://docs.python.org/2/library/random.html#random.choice.

**Comment by Linkid** [comment]
Ok, I found what is it : it worked when [random](https://github.com/fofix/fofix/blob/dedc1f800452cbf4f1bf43fa820b620ccc4cc311/fofix/game/guitarscene/GuitarScene.py#L39) was at the end of the import block. So, another module import it… I will fix that.

**Comment by Linkid** [comment]
This should be fixed with #200.

**Comment by AEFeinstein** [comment]
Thanks!
